.github/workflows/pull-request-workflow.yml

```diff
@@ -27,5 +27,5 @@ jobs:
     uses: ./.github/workflows/canary-release.yml
     secrets:
       githubToken: ${{ secrets.GITHUB_TOKEN }}
-      npmToken: ${{ secrets.NPM_PUBLISH_TOKEN }}
+      npmToken: ${{ secrets.NPM_CHROMATIC_COM_PUBLISH_TOKEN }}
```

Comment:
Would it make sense to remove the `_COM_` from the secret name since it's a Chromatic publish token?